### PR TITLE
[WIP] Add Datepicker initial draft

### PR DESCRIPTION
**Pull request by pompomon**

### CODE REVIEW

**Comment by gregwhitworth** [comment]
I've assigned it to @chrisdholt @nicholasrice in addition to myself. Chris/Nic it doesn't require both of you but I'd like someone outside of myself to review it.

**Comment by gregwhitworth** [comment]
I've removed the other reviewers as I don't think this PR in its current form requires an additional review

**Comment by pompomon** [comment]
Thank you for all the comments @gregwhitworth !
I agree with the proposal to split this PR, I'll prepare the analysis only part next week (unfortunately, I won't have time to do that earlier).
Please what would recommend - to close this PR entirely and open a new one just with analysis, or  remove non-analysis parts from this PR and keep it going here?

**Comment by gregwhitworth** [comment]
@pompomon whichever you prefer, I have no strong preference.

File: research/src/pages/datepicker.research.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import Anatomy from '../components/anatomy'
+import Concepts from '../components/concepts'
+import ComponentCoverage from '../components/component-coverage'
+
+<ComponentCoverage component="Datepicker" />
+
+## Concepts
+
+<Concepts component="Datepicker" />
```
[the remark this answers is not in the document]
Comment (pompomon) [reply research/src/pages/datepicker.research.mdx:15]: @gregwhitworth , please could you elaborate on that a bit? I don't have those concepts like `id` or `theme` in the current analysis (datepicker.research page), do you mean that those need to be added?

**Comment by pompomon** [comment]
I've remove the proposal part from this PR and would proceed with analysis here. Would keep the comments to the proposal part open, so I can return to them later

**Comment by gregwhitworth** [comment]
@pompomon thanks for splitting this out. I went ahead and made a few tweaks at you didn't have a menu category defined (so your research wasn't showing up in the nav) and removed the path to the proposal that doesn't exist.

**Pull request merged**